wiki/osu!wiki_Contribution_Guide/Locally/en.md

```diff
@@ -0,0 +1,71 @@
+# Locally
+
+*Main page: [osu!wiki Contribution Guide](/wiki/owcg).*
+
+*This article continues from the main page* and assumes that you will be using [GitHub Desktop](https://desktop.github.com). You may use other git clients on your own terms. Even if you do work locally, you will still need to access GitHub to create pull requests to make your changes happen.
+
+## Installing GitHub Desktop
+
+You will need to then install [GitHub Desktop](https://desktop.github.com). See [Getting Started with GitHub Desktop](https://help.github.com/desktop-classic/guides/getting-started/) for help.
```

Comment:
you need to install

🤔

Reply:
the sentence assumes that you have read the main page... will reword to your suggestion